code/src/Components/FilmList.jsx

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styled from 'styled-components'
+
+const FilmList = ({films}) => {
+    return(
+        <Main>
+        {films.map((film) => {
+        return(
+        <Link to={`/movies/${film.id}`} key={film.id}>
+        <Filmcard>
+            <FilmImageGroup>
+                <CardImage className="card-image" src={`https://image.tmdb.org/t/p/w500${film.poster_path}`} alt={film.title} />
+                <FilmImageOverlay></FilmImageOverlay>
+                <FilmTextOverlay>
+                    <FilmTitle>{film.title}</FilmTitle>
+                    <h3>Released {film.release_date}</h3>
+                </FilmTextOverlay>
+            </FilmImageGroup>
+        </Filmcard>
+        </Link>
+        )
+        })}
+        </Main>
+    )
+}
+
+export default FilmList
+
+// Styled Components
```

Comment:
You did a wonderful job putting together the different stylings in this component. We learn a lot reading this page and how you guys apply the styling in your return code!! Standing ovation!